### ISSUE TITLE
Attention Loss (smooth loss in the code) without square?

### ISSUE THREAD

**Issue by ewrfcas** [albertpumarola/GANimation#75]
Hi,
I find that the smooth loss (Attention Loss) (include its l2 decay) is simply calculated by torch.mean and torch.sum without square, which is different from the formula in the paper. What's the meaning of doing this?

**Comment by liyu10000** [comment]
> Hi,
> I find that the smooth loss (Attention Loss) (include its l2 decay) is simply calculated by torch.mean and torch.sum without square, which is different from the formula in the paper. What's the meaning of doing this?

Hi,

It may be too late to ask. Have you tried to use squared loss in training? I am curious if that could get to better performance.

**Comment by ewrfcas** [comment]
The loss without square is enough for this model.
